app/serializers/rest/admin/account_serializer.rb

```diff
@@ -83,4 +83,8 @@ def ips
   def ip
     ips&.first&.ip
   end
+
+  def media_attachments
+    object.media_attachments.sum(:file_file_size)
```

Comment:
I'm a bit concerned about the performance implications: this is a rather expensive request, and having it in the serializer in this way hides its costs. In practice, it's also used when *listing* accounts so it can end up being fairly expensive.

Reply:
What would be a better way to get this data?

Reply:
Or is there a way to override the code to only display on single account api endpoints? I am not a good enough RoR dev to know off hand, in python that is what I would do if load tests showed full lists would fail, but I don't know if that is possible or a best practice here. thanks.